frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx

```diff
@@ -38,63 +97,112 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs, appName }) => {
-                          <div className="odc-gitops-details__env-section__url-empty-state">
+                          <div className="odc-gitops-details__env-section__cluster-url-empty-state">
                             {t('gitops-plugin~Cluster URL not available')}
                           </div>
                         )}
                       </StackItem>
-                      <StackItem className="co-truncate co-nowrap">
-                        <span className="co-resource-item odc-gitops-details__env-section__co-resource-item">
-                          <ResourceIcon kind="Project" />
-                          <Label className="co-resource-item__resource-name" isTruncated>
-                            {env.environment}
-                          </Label>
-                        </span>
-                      </StackItem>
-                      {env.environment && argocdLink && (
-                        <StackItem className="co-truncate co-nowrap">
-                          <ExternalLink
-                            href={`${argocdLink.spec.href}/applications/${env.environment}-${appName}`}
-                            text="Argo CD"
-                            additionalClassName="odc-gitops-details__env-section__argocd-link"
-                          />
+                      {env.status && (
+                        <StackItem className="odc-gitops-details__env-section__status-label">
+                          <Tooltip content="Sync status">
+                            <span>{renderStatusLabel(env.status)}</span>
+                          </Tooltip>
+                        </StackItem>
+                      )}
+                    </Stack>
+                  </CardTitle>
+                  <CardBody>
+                    <Stack className="odc-gitops-details__revision">
+                      {env.revision ? (
+                        <>
+                          {env.revision?.message && (
+                            <StackItem className="odc-gitops-details__revision__message">
+                              {t('gitops-plugin~{{message}}', { message: env.revision.message })}
+                            </StackItem>
+                          )}
+                          <StackItem className="odc-gitops-details__revision__author-sha">
+                            {env.revision?.author && (
+                              <span className="odc-gitops-details__revision__author">
+                                {t('gitops-plugin~by {{author}}', { author: env.revision.author })}{' '}
+                              </span>
+                            )}
+                            {env.revision?.revision && (
+                              <Label
+                                className="odc-gitops-details__revision__sha"
+                                color="blue"
+                                icon={routeDecoratorIcon(
+                                  'https://www.github.com/reginapizza/taxi',
```

Comment:
Hmmm... Don't think you need to send your specific repo. Maybe use `console` if `github.com` doesn't work by itself.

Reply:
We originally meant to use the repo url for this, my github repo was supposed to just be a placeholder. Since we're not getting that data on this page, I'm removing routeDecoratorIcon temporarily and putting in `<GitAltIcon />`  until we can get it.